src/java/devcontainer-feature.json

```diff
@@ -33,10 +33,20 @@
       "default": false,
       "description": "Install Gradle, a build automation tool for multi-language software development"
     },
+    "gradleVersion": {
```

Comment:
Can we bump the minor version in here as we are adding a new `option` ( ie `1.1.0`)?https://github.com/devcontainers/features/blob/a15440b229a8a78fbd64c97ba0a7645345c58c90/src/java/devcontainer-feature.json#L3

The Java Feature will automatically get published once this PR is merged. (Provided the version has changed)